_elements/tables.md

```diff
@@ -5,8 +5,95 @@ title: Tables
 ---
 
 <div class="preview">
-  <!-- Add HTML markup for example here -->
-  <img src="{{ site.baseurl }}/assets/img/static/Tables_UI_v1.png">
+
+  <h3>Bordered Table</h3>
+
+  <table class="usa-table-bordered">
```

Comment:
Yeah it will look like this:

<img width="862" alt="screen shot 2015-07-30 at 10 29 49 am" src="https://cloud.githubusercontent.com/assets/5249443/8990222/b52fc830-36a5-11e5-810e-e9cd53af9ea5.png">